src/cobra/core/reaction.py

```diff
@@ -781,20 +886,32 @@ def get_coefficient(self, metabolite_id):
         _id_to_metabolites = {m.id: m for m in self._metabolites}
         return self._metabolites[_id_to_metabolites[metabolite_id]]
 
-    def get_coefficients(self, metabolite_ids):
-        """
-        Return the stoichiometric coefficients for a list of metabolites.
+    def get_coefficients(
+        self, metabolite_ids: Iterable[Union[str, Metabolite]]
+    ) -> Iterator[Union[int, float]]:
+        """Return the stoichiometric coefficients for a list of metabolites.
 
         Parameters
         ----------
         metabolite_ids : iterable
             Containing ``str`` or ``cobra.Metabolite``s.
 
+        Returns
+        -------
+        map: Iterable
+            Returns the result of map function, which is a map object (an Iterable).
+
         """
         return map(self.get_coefficient, metabolite_ids)
 
-    def add_metabolites(self, metabolites_to_add, combine=True, reversibly=True):
+    def add_metabolites(
+        self,
+        metabolites_to_add: Dict[Metabolite, Union[int, float]],
+        combine: Optional[bool] = True,
```

Comment:
Please make the type as `bool` and document the optional behaviour in the docstring.